CHANGELOG.md

```diff
@@ -1,5 +1,56 @@
 # Change Log
 
+## Version 3.2.0 *(2020-04-02)*
+
+This is a new minor release with numerous internal changes.
+
+### Summary of changes
+
+* Chucker won't load the whole response into memory anymore, but will mutlicast it with the help of temporary files. It allows to avoid issues with OOM, like in this [issue](https://github.com/ChuckerTeam/chucker/issues/218).
+This change also allows to avoid problems with Chucker consuming responses, like reported in this [issue](https://github.com/ChuckerTeam/chucker/issues/242).
+* Added a red open padlock icon to clearly indicate HTTP requests in transactions list.
+* Added TLS info (version and cipher suite) into `Overview` tab.
+* Added ability to encode/decode URLs.
+* Added RTL support.
+* Switched from AsyncTasks to Kotlin coroutines.
+* Switched from Travis CI to Github Actions.
+* Switched to ViewBinding.
+* Bumped targetSDK to 29.
+* Greatly increased test coverage (we will add exact numbers and reports pretty soon).
+
+### Bugfixes
+
+* Fixed an [issue](https://github.com/ChuckerTeam/chucker/issues/218) with OOM exceptions on big responses.
+* Fixed an [issue](https://github.com/ChuckerTeam/chucker/issues/242) with Chucker throwing exception when used as `networkInterceptor()`.
+* Fixed an [issue](https://github.com/ChuckerTeam/chucker/pull/240) with exception when DexGuard is used.
+* Fixed an [issue](https://github.com/ChuckerTeam/chucker/issues/254) with response search being case-sensitive.
+* Fixed an [issue](https://github.com/ChuckerTeam/chucker/issues/255) with missing search icon on Response tab.
+* Fixed an [issue](https://github.com/ChuckerTeam/chucker/issues/241) with overlapping texts.
+
+### Dependency updates
+
+* Updated Kotlin to 1.3.71
```

Comment:
We should mention that we now depend on `kotlinx-coroutines-core` and `kotlinx-coroutines-andriod` + the version